css/pages/index.css

```diff
@@ -3,50 +3,122 @@
 }
 
 .container {
-  width: 750px;
+  width: 100%;
   text-align: center;
   font-size: 1.5rem;
 }
 
+.second-screen {
+  width: 100%;
+}
+
 .cards-title {
-  margin-bottom: 32px;
-  color: #e20665;
+  margin-top: 1041px;
+  margin-bottom: 60px;
+  width: 100%;
+}
+
+h3 {
+  color: #091133;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  letter-spacing: 10%;
+  font-weight: 600;
+  font-size: 250%;
+  line-height: 66%;
+  font-style: normal;
+  font-family: 'raleway';
 }
 
 .cards {
   display: flex;
-  justify-content: space-around;
+  justify-content: center;
+  width: 100%;
+  margin-left: 5%;
 }
 
 .card {
   border: 1px solid #f3efef;
-  box-shadow: 0 0 15px -9px rgba(0, 0, 0, 0.65);
-  border-radius: 2px;
-  width: 30%;
-  padding: 18px 15px;
+  border-radius: 18px;
   display: flex;
   justify-content: center;
   align-items: center;
   font-weight: 600;
   font-size: 0.8em;
   text-align: center;
-  color: #000000;
+  letter-spacing: 1px;
+  background-color: #e30062;
+  color: #ffffff;
+  font-family: 'Roboto';
+  font-style: normal;
+  padding: 3%;
+  margin-right: 7%;
+}
+
+#card1 {
+  width: 18%;
+}
+
+#card2 {
+  width: 18%;
+}
+
+#card3 {
+  width: 18%;
```

Comment:
What's the use of these id's if they're having the same properties? Why not use classes for the same?